src/plugins/theme/fontFamily.js

```diff
@@ -5,13 +5,15 @@ module.exports = function () {
   return {
     mono: ['"Roboto Mono"', 'Menlo', '"Courier New"', 'monospace'],
     sans: [
+      '"Source Sans 3"',
```

Comment:
Adding the SS3, but leaving the SS Pro as 2nd place for now

Reply:
Why?

Reply:
Also, 
Don't we need to update the import as well?

https://github.com/SU-SWS/decanter/blob/v7/src/css/index.css#L4

Reply:
Just in case someone installs this latest version of Decanter but missing the note about the font change in the projects. They have to also update the font import in their own projects, or else it will render the other fall back font like Arial 😬 

It might not be obvious when they preview locally because the serif and sans fonts are not super different from the default fonts (times new romans or arial etc)

Open to suggestions though 😄

Reply:
I don't think it hurts to have the fallback and that makes sense with what you are saying on the expectation of the project importing the fonts.